mentoring/components/templates/html/mcqblock.html

```diff
@@ -0,0 +1,19 @@
+<fieldset class="choices questionnaire">
+  <legend class="question">
+    {% if not hide_header %}<h3 class="question-title">{{ self.display_name }}</h3>{% endif %}
+    <p>{{ self.question }}</p>
+  </legend>
+  <div class="choices-list">
+    {% for choice in custom_choices %}
+    <div class="choice">
+      <div class="choice-result fa icon-2x"></div>
```

Comment:
Tips and checkboxes currently don't seem to appear at the right place:

![screenshot-sandbox opencraft com 2015-02-24 14-21-37](https://cloud.githubusercontent.com/assets/514483/6350039/867d71d0-bc30-11e4-82df-1fd13b472160.png)


Ie, the checkbox/checkmark should appear next to the answer, not above next to the question. 

The tip doesn't seem to be displayable either - I had one set on "yes" which didn't appear.

![screenshot-sandbox opencraft com 18010 2015-02-24 14-20-26](https://cloud.githubusercontent.com/assets/514483/6350033/7d893e56-bc30-11e4-8bf4-a20e48daeb99.png)